javascript/javascript1/week1/dog age calculator.js

```diff
@@ -0,0 +1,11 @@
+const dogYearOfBirth = 2016;
+let dogFutureYear = 2045;
+const dogYear = 6;
+let result = dogFutureYear- dogYearOfBirth;
+let shouldShowResultInDogYear = true;
+
+if (shouldShowResultInDogYear) {
+    console.log("Your dog will be " +result * dogYear + " dog years old in " + dogFutureYear)
+} else {
+    console.log("our dog will be " + result + " human years old in " + dogFutureYear)
+}
```

Comment:
Well done Trish! Just a few things. 

Since you are not changing dogFutureYear in your code, you could make that a const instead - and again, rather than writing all of those qutation marks and plusses, you could use string interpolation as above, e.g. ``Your dog will be ${result * dogYear} dog years old in ${dogFutureYear}``